### DISCUSSION TITLE
Awesomeness!

### DISCUSSION THREAD

**Discussion by AndrewEQ** [NekitCorp/chrome-extension-svelte-typescript-boilerplate#18] [General]
Great project man,

Its been awhile since I have had a look at Chrome Extensions and now you've combined all my favourites, awesome job and keep up the good work! 😊

P.s. I came across this guys tailwindcss integration if you wanna add it to your boilerplate or start another called "chrome-extension-svelte-typescript-tailwindcss-vite" perhaps? 😄 https://carlogino.com/blog/svelte-chromium-extension

**Comment by NekitCorp** [comment]
Hi, thanks a lot for your feedback! 😊 The article looks pretty interesting. Regarding the new project, it looks like the author of the article has already made such a boilerplate – https://github.com/codegino/svelte-tailwind-chrome-plugin/

**Reply by AndrewEQ** [reply]
I believe yours is a more comprehensive boilerplate, just missing tailwindcss then.

**Reply by areeeeb** [reply]
Hello, if I were to add tailwindcss in your boilerplate, do I just have to follow the steps regarding tailwindcss in the above tutorial? Would that be enough?

**Reply by NekitCorp** [reply]
@areeeeb Hello! Yes, the tutorial above should be enough. But, I also added an example of using Tailwind CSS with the current boilerplate in a separate branch: https://github.com/NekitCorp/chrome-extension-svelte-typescript-boilerplate/tree/tailwindcss, you can use it as a starting point.
